symengine/cwrapper.cpp

```diff
@@ -280,6 +281,12 @@ int number_is_complex(const basic s)
     return (int)((down_cast<const Number &>(*(s->m))).is_complex());
 }
 
+int basic_has_symbol(const basic e, const basic s)
+{
+    //Should have a check for s being a symbol
+    return (int)(has_symbol(*(e->m), *(s->m)));
+}
```

Comment:
@isuruf, I think we don't need to check that `s` is a Symbol here, do we? 

The implementation is here: https://github.com/symengine/symengine/blob/6da98cf4527802c8c98ddc696dfd2999765cda1b/symengine/visitor.h#L93, and it accepts `Basic`, and apparently it can be either a Symbol or a FunctionSymbol. And if it is neither, it seems it "works", it will just return `false` I think.

Reply:
Yes, it will just return false.